news/37.feature

```diff
@@ -0,0 +1,2 @@
+Drop Python 2.7 and 3.7 support. Support only python >3.8, Plone 5.2 and Plone 6.
```

Comment:
I think you forgot to mention removing the settings.

I also think this is a `breaking`, not a `feature`.